packages/core/src/locales/be.ts

```diff
@@ -0,0 +1,124 @@
+import type { $ZodStringFormats } from "../checks.js";
+import type * as errors from "../errors.js";
+import * as util from "../util.js";
+
+const Sizable: Record<string, { unit: string; verb: string }> = {
+  string: { unit: "сімвалаў", verb: "мець" },
+  file: { unit: "байтаў", verb: "мець" },
+  array: { unit: "элементаў", verb: "мець" },
+  set: { unit: "элементаў", verb: "мець" },
+};
+
+function getSizing(origin: string): { unit: string; verb: string } | null {
+  return Sizable[origin] ?? null;
+}
```

Comment:
btw are plural forms correctly handled in this PR?

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/8860b066-5ce1-4cf1-baae-0191112f10a6" />